### PR TITLE
Added a new test

### PR DESCRIPTION
**Pull request by talgoldy**

### CODE REVIEW

**Review by amitzur — CHANGES_REQUESTED** [review]
It's not clear what issue is surfaced from this test. I personally know what this is related to, but `https://demo.applitools.com` is maintained by the marketing team, it can change without our control, and we don't have the source code for it.

Please create a static page in the `demo` repo that reproduces this scenario.

File: coverage-tests.js [hunk 1]
```diff
@@ -1604,6 +1604,23 @@ test('check window fully with html scrollRootElement after scroll', {
   }
 })
 
+test('check window fully with html scrollRootElement after scroll when fail to scroll', {
+  variants: {
+    'with css stitching': {config: {stitchMode: 'CSS', baselineName: 'TestCheckWindowFullyWithHtmlScrollRootElementAfterScroll'}},
+    'with scroll stitching': {config: {stitchMode: 'Scroll', baselineName: 'TestCheckWindowFullyWithHtmlScrollRootElementAfterScroll_Scroll'}},
+  },
+  test({driver, eyes}) {
+    driver.visit('https://demo.applitools.com')
```
Comment (kyryloonufriiev) [inline coverage-tests.js:1613]: I think a better practice will be to add this page to the list of named pages, with a proper name (what is special about this page?).

**Review by kyryloonufriiev — CHANGES_REQUESTED** [review]
I agree with Amit. It is not clear what this test for.

**Comment by amitzur** [comment]
reference: https://trello.com/c/4NDa6Vxl/2416-stitching-issue-at-the-bottom-of-the-page-java-datorama

File: coverage-tests.js [hunk 2]
```diff
@@ -1604,6 +1604,23 @@ test('check window fully with html scrollRootElement after scroll', {
   }
 })
 
+test('check window fully with html scrollRootElement after scroll when fail to scroll', {
+  page: 'Demo',
```
Comment (kyryloonufriiev) [inline coverage-tests.js:1608]: I don't think the name `Demo` makes it more clear than it was. Why this page? If it doesn't matter let's use the `Default`, but probably it matters and I would like to get some information about the page from its name.

**Pull request merged**